airflow/www/static/js/dag/nav/FilterBar.tsx

```diff
@@ -152,6 +208,7 @@ const FilterBar = () => {
             background="white"
             variant="outline"
             onClick={clearFilters}
+            disabled={areFiltersDefault}
```

Comment:
The "filter tasks" option on the right panel was influencing the state of the filter button - when tasks are filtered there, the "clear filters" button gets active, but clicking on it does not reset these. And as such the click does not have an effect.

Reply:
Fixed!